charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.39.1
-version: 15.16.1
+version: 15.16.2
```

Comment:
new feature should raise at leasdt minor version.
as it seems defaults are changed it would be a major version change.

Reply:
@monotek Done